README.md

```diff
@@ -379,6 +379,112 @@ parameters.completionBlockQueue = dispatch_get_main_queue();
 }];
 ```
 
+### Multiple Accounts Mode
+
+MSAL also provides a public API to query multiple accounts, granted that they exist in the MSAL cache.
+
+1) Make sure the umbrella header MSAL/MSAL.h is imported 
+
+2) Create config, then use it to initialize an application object 
+
+3) Also initialize MSALAccountEnumerationParameters object with the account identifier. Each MSALAccount object has a parameter called "identifier", which represents the unique account identifier associated with the given MSALAccount object. We recommend using it as the primary search criterion. 
+
+4) Then invoke the API "accountsFromDeviceForParameters" from the application object using the enumeration parameter. If you have multiple accounts in MSAL cache, it will return an array containg MSALAccounts that have the account identifier you specified in the previous step. 
+
+5) Once the MSAL account is retrieved, invoke acquire token silent operation
+
+#### Swift
+
+```swift
+#import MSAL //Make sure to import MSAL  
+
+let config = MSALPublicClientApplicationConfig(clientId:clientId
+                                           	redirectUri:redirectUri
+                                            	authority:authority)
+guard let application = MSALPublicClientApplication(configuration: config) else { return }
+
+let accountIdentifier = "9f4880d8-80ba-4c40-97bc-f7a23c703084.f645ad92-e38d-4d1a-b510-d1b09a74a8ca"
+let parameters = MSALAccountEnumerationParameters(identifier:accountIdentifier)
+
+var scopeArr = ["https://graph.microsoft.com/.default"]
+
+if #available(iOS 13.0, *)
```

Comment:
Is this API only available for iOS?

Reply:
Good catch - this API is also available on MacOS 10.15. I overlooked it since the customer that initially reported this issue had an iOS set up. Thank you so much for catching it! :)